src/generators/css-filter.ts

```diff
@@ -22,7 +21,7 @@ export enum FilterMode {
 export function hasChromiumIssue501582() {
```

Comment:
Please adjust the comment that is above this function, as I actually now realizes I didn't got the function name right as it was being used for "hasPatchForChromiumIssue501582" was a better name.